.gitignore

```diff
@@ -1,3 +1,16 @@
+# Editors
+.vscode/
+
+# dynamically generated docs
+docs/source/components.rst
+
+#
+# The above is custom entries for the labscript suite project
+#
+# The below is the standard GitHub Python .gitignore list. This contains no custom
```

Comment:
There is at least one modification to the standard GitHub Python .gitignore below this line:
```gitignore
# Sphinx documentation
docs/html/
```

Reply:
Oh did we customise that...oops! I'll fix